apps/content_types/content_types.py

```diff
@@ -111,6 +109,9 @@ def on_create(self, docs):
         for doc in docs:
             self._set_updated_by(doc)
             self._set_created_by(doc)
+            if 'schema' not in doc and 'editor' not in doc:
+                doc['schema'] = DEFAULT_SCHEMA
+                doc['editor'] = DEFAULT_EDITOR
```

Comment:
This is used to make distinction between prepopulation and creation of new content item from UI.
For prepopulation the definition of content type is complete so default is not needed
For add new content type from GUI is created a content type by specify only name and description.
Also editor and schema are correlated so if one is set also the other should be set.